components/AddFundsSourcePicker.js

```diff
@@ -56,28 +56,18 @@ class AddFundsSourcePickerForUser extends React.Component {
     this.state = { loading: true };
   }
 
-  onChange = async e => {
-    this.props.onChange(e.target.value);
-  };
-
   render() {
     const hosts = this.props.LoggedInUser.hostsUserIsAdminOf();
+    const hostOptions = [
+      { key: 'addfsph-00', value: '' },
+      ...hosts.map(h => {
+        const value = h.legacyId || h.id;
+        return { key: `addfsph-${h.id}`, value: value, label: h.name };
+      }),
+    ];
     return (
       <div>
-        <FormControl
-          id="sourcePicker"
-          name="template"
-          componentClass="select"
-          placeholder="select"
-          onChange={this.onChange}
-        >
-          <option value="" key="addfsph-00" />
-          {hosts.map(h => (
-            <option value={h.legacyId || h.id} key={`addfsph-${h.id}`}>
-              {h.name}
-            </option>
-          ))}
-        </FormControl>
+        <StyledSelect id="sourcePicker" placeholder="select" options={hostOptions} onChange={this.props.onChange} />
```

Comment:
Maybe we could title case the placeholder. Also, this empty entry doesn't make much sense since the field is required, I think we should remove it. Finally, we should put `isSearchable={false}` on the select since there are only a few entries.

![image](https://user-images.githubusercontent.com/1556356/104162552-1dfaca00-53f5-11eb-9c19-019207152ee5.png)

Reply:
@Betree : Thanks. I've corrected this. Please take a look and let me know if you see any other issues. 👍🏽